mods/ra/maps/soviet-01/rules.yaml

```diff
@@ -35,8 +40,7 @@ AFLD:
 	AirstrikePower@spyplane:
 		Prerequisites: ~disabled
 	ParatroopersPower@paratroopers:
-		ChargeInterval: 1500
-		DropItems: E1,E1,E1,E2,E2
+		Prerequisites: -disabled
```

Comment:
I just saw this now, this likely wants to be `~` instead?

Reply:
🤦 Updated